<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckBrd_ai</class>
 <widget class="QWidget" name="CheckBrd_ai">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/paimeng1.jpg</normaloff>:/image/paimeng1.jpg</iconset>
  </property>
  <widget class="QLineEdit" name="player1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>620</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>玩家1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>630</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poor Richard</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>240</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>240</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RenShu">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>认输</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player2_img">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>520</y>
     <width>93</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/white.png</normaloff>:/image/white.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Regame">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>360</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="player1_img">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>520</y>
     <width>93</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/black.png</normaloff>:/image/black.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="player2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>620</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>玩家2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>倒计时</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Zongshoushu">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>130</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Tuichu">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>20</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Count">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>130</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>60</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>总手数</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ai_enable_3">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>480</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>落子前确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_black">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>710</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="help_white">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>710</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="AIblack">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>470</y>
     <width>118</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>黑AI</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="AIwhite">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>480</y>
     <width>118</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>白AI</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
